app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,67 @@
+module SessionsHelper
```

Comment:
Aren't session helpers included from devise by default?

Reply:
not using devise